tests/test_ingestion.py

```diff
@@ -45,13 +47,7 @@ def setUpClass(cls):
             message = "obs_test not setup. Skipping."
             raise unittest.SkipTest(message)
 
-        obsDir = os.path.join(getPackageDir('obs_test'), 'config')
-        cls.config = ingestion.DatasetIngestConfig()
-        cls.config.dataIngester.load(os.path.join(obsDir, 'ingest.py'))
-        cls.config.calibIngester.load(os.path.join(obsDir, 'ingestCalibs.py'))
-        cls.config.defectIngester.load(os.path.join(obsDir, 'ingestCalibs.py'))
-        # The real obs_test can't ingest defects because they're hardcoded
-        cls.config.defectIngester.register.tables.append('defect')
+        cls.config = cls.makeTestConfig()
```

Comment:
Because the config is a) immutable and b) expensive to construct. (In fact, the two slowest unit tests in `ap_verify` are slow precisely because they can't avoid making a real config.)

Reply:
Those are good reasons.

The reason to put most things in `setUp` is so that they get reset for every test, in case a test (intentionally or accidentally) changes them.